Packs/SymantecEDR/Integrations/SymantecEDR/SymantecEDR.py

```diff
@@ -0,0 +1,2734 @@
+        """
+        Client method for Get File Entity for specific SHA2
+        Args:
+            sha2 (str): File SHA256
+        Returns:
+            return response json
+        """
+        return self.http_request(
+            method='GET',
+            endpoint=f'/atpapi/v2/entities/files/{sha2}',
+            params={},
+            json_data={}).json()
+
+    def get_sandbox_status(self, command_id: str) -> dict[str, Any]:
+        """
+        Client method to Get Sanbox Status for specific file
+        Args:
+            command_id (str): sandbox command ID
+        Returns:
+            return response json
+        """
+        return self.http_request(
+            method='GET',
+            endpoint=f'/atpapi/v2/sandbox/commands/{command_id}',
+            params={},
+            json_data={}).json()
+
+    def submit_file_to_sandbox_analyze(self, payload: dict) -> dict[str, Any]:
+        """
+        Client method for sandbox analyze
+        Args:
+            payload: request body
+
+        Returns:
+            return response json
+        """
+        return self.http_request(
+            method='POST',
+            endpoint='/atpapi/v2/sandbox/commands',
+            params={},
+            json_data=payload
+        ).json()
+
+
+''' HELPER FUNCTIONS '''
+
+
+def convert_to_iso8601(timestamp: str) -> str:
+    """ Convert timestamp from iso 8601 format
+
+    Args:
+        timestamp: Any valid timestamp or provide timedelta e.g. now, "<n> days", "<n> weeks",
+        "<n> months", "1 months ago"
+
+    Returns: return timestamp in iso 8601 format.
+
+    """
+    if datetime_from_timestamp := dateparser.parse(timestamp, settings={'TIMEZONE': 'UTC'}):
+        return f'{datetime_from_timestamp.strftime(ISO8601_DATE_FORMAT)[:23]}Z'
+    else:
+        raise ValueError(f'{timestamp} could not be parsed')
+
+
+def extract_headers_for_readable_output(summary_data: list[dict]) -> list:
+    """
+    Symantec EDR formatting Readable output Header
+    Args:
+        summary_data (list[dict]): Human readable output summary data
+
+    Returns:
+        Return string headers to camel case.
+
+    """
+    if not summary_data:
+        raise DemistoException('No Readable output data found to display.')
+
+    headers = summary_data[0] if summary_data else {}
+    headers = list(headers.keys())
+    return [camelize_string(column) for column in headers]
+
+
+def get_data_of_current_page(response_data: list[dict[str, Any]], offset: int = 0, limit: int = 0) -> list:
+    """
+    Retrieve list element based on offset and limit
+    Args:
+        response_data (list): Raw API result list
+        offset (int) : Offset
+        limit (int) : Page Limit
+
+    Returns:
+        Return List of object from the response according to the limit, page and page_size.
+
+    """
+
+    if offset >= 0 and limit >= 0:
+        demisto.debug(f'I am here {offset} <=> {limit} ...')
```

Comment:
```suggestion
        demisto.debug(f'{offset=}, {limit=}')
```

Reply:
I will remove this line as it was added during testing as no relevant to add as LOG/Debug.